### PR TITLE
Update resources.json

### PR DESCRIPTION
**Pull request by Monodeep-Das**

### CODE REVIEW

**Review by TenTraicion — APPROVED** [review]

**Comment by Monodeep-Das** [comment]
Hey @TenTraicion can you please accept my PR

File: src/assets/data/resources.json [hunk 1]
```diff
@@ -6,6 +6,12 @@
         "url": "https://undraw.co/",
         "tags": ["design", "ilustration"]
     },
+    {
+        "title": "Iconic Pack",
```
Comment (clebsonf) [inline src/assets/data/resources.json:10]: duplicate resource https://github.com/OctobearHub/octobear.org/blob/12e90b79cd9533f4c7c8c0b7b078c0eb7a2a625a/src/assets/data/resources.json#L10-L13

**Review by clebsonf — CHANGES_REQUESTED** [review]
Hi @Monodeep-Das, how are you? Looking around, I realized that we already had this resource. Unfortunately, I won't be able to accept your pull request, but if you wish, you can create another one with some other resource.

**Comment by TenTraicion** [comment]
@Monodeep-Das, it's been 9 days since we requested a change. Since there are no activities and reply from you we are closing this PR. You are welcome to add another resource and open a new PR. Thanks.

**Pull request closed without merging**